cmake/therock_subproject.cmake

```diff
@@ -20,6 +20,8 @@ set_property(GLOBAL PROPERTY THEROCK_DEFAULT_CMAKE_VARS
   THEROCK_BINARY_DIR
   THEROCK_BUILD_TESTING
   THEROCK_USE_SAFE_DEPENDENCY_PROVIDER
+  THEROCK_ROCM_LIBRARIES_SOURCE_DIR
+  THEROCK_ROCM_SYSTEMS_SOURCE_DIR
```

Comment:
nit: maybe sort these next to `THEROCK_SOURCE_DIR` and `THEROCK_BINARY_DIR`? Either after `THEROCK_BINARY_DIR` or between them.